tools/debugger/tests/do_exception.expected

```diff
@@ -3,7 +3,7 @@ Connection created!!!
 Stopped at tools/debugger/tests/do_exception.js:28
 (escargot-debugger) c
 Exception: RangeError: Test exception
-tools/debugger/tests/do_exception.js:21:10 [depth:0] (in g)
-tools/debugger/tests/do_exception.js:25:4 [depth:1] (in f)
-tools/debugger/tests/do_exception.js:28:1 [depth:2]
+tools/debugger/tests/do_exception.js:21:10 (in g)
+tools/debugger/tests/do_exception.js:25:4 (in f)
+tools/debugger/tests/do_exception.js:28:1
```

Comment:
Depth information is deleted. Is it intended?

Reply:
That was intentional. I realized you cannot get the scope for a backtrace, since you are not in breakpoint mode. So this has never been worked. For me this is similar to async backtrace recording (btw we have checked, and v8 also does not save any scope for a recorded backtrace)